ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml

```diff
@@ -13,17 +85,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests
-  reporter_config:
```

Comment:
let's add this back

Reply:
name: periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests is now part of the list, the auto config generation seems to somehow added arm in place of the existing one and added a new config for periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

Reply:
See my comment in the config file about this that it wont replace but add a new item to the list

Reply:
Think you'll need to regenerate these files using 'make jobs'